house/forms.py

```diff
@@ -0,0 +1,23 @@
+
+from django import forms
+
+from house.models import House, Job
+
+
+class HouseForm(forms.ModelForm):
+    class Meta:
+        model = House
+        fields = ('country', 'city', 'children')
+
+    jobs = Job.choices.copy()
+    jobs.append((None, '---------'))
```

Comment:
Why not add that option to the choises enum?

Reply:
what is the difference between job and  parent_profession_1/parent_profession_1?